### DISCUSSION TITLE
OIDC - how to retrieve content inside JWT CLAIM

### DISCUSSION THREAD

**Discussion by reichert** [Budibase/budibase#6236] [Help]
Hi guys,
I've sucessfully configured BUDIBASE authentication through OIDC against a KEYCLOAK server.

The JWT issued by KEYCLOAK server has some fields (which is called CLAIMs) and I'd like to acess it in my application for logic / show to user purpose.

1) Take a look at the bindings available below in my application. They where propagated from KEYCLOAK to the "Current User" successfully:
![image](https://user-images.githubusercontent.com/1287032/172634942-e66ec885-cdff-433a-be21-3d622ef2fe87.png)

The question is:
a) how can I access the element through handlebars or javascript (example: `{{ Current User.thirdPartyProvider.given_name }}`) inside a paragraph element or any other place where javascript can be used?

b) I tried to print the "Current User" whole object without success in a paragraph element (but I can print the "Current User.firstName"). How can I do that?

Thanks!

**Comment by melohagan** [comment]
Hi @reichert 

This issue may be related: https://github.com/Budibase/budibase/issues/3827

I did try accessing Current User, and as you stated no luck. It may be purposefully designed this way so as to not expose the properties of the object, because no other object in Budibase behaves that way off the top of my head.

With that said, your binding looks wrong, should you not have tried `thirdPartyProfile` - not provider. I don't think that will work anyway though.
